treeherder/client/thclient/client.py

```diff
@@ -654,6 +654,8 @@ class TreeherderClient(object):
 
     UPDATE_ENDPOINT = 'job-log-url/{}/update_parse_status'
 
+    API_ROOT = "https://treeherder.mozilla.org/api/project"
```

Comment:
You should construct this from the class members for protocol and host. You can use the _get_uri method for this: https://github.com/mozilla/treeherder/blob/master/treeherder/client/thclient/client.py#L674